packages/jwt/src/test/test-utils.ts

```diff
@@ -0,0 +1,46 @@
+import type { jwtClaimsTestCases } from "../generated/jwt-claims-test-cases";
+import type { JWTPayload } from "../types";
+import { createAkashAddress } from "./seeders/akash-address.seeder";
+
+/**
+ * Replaces template values in JWT test cases with actual values
+ *
+ * Supports the following template patterns:
+ * - {{.Issuer}} - Replaced with a generated Akash address for the issuer
+ * - {{.Provider}} - Replaced with a generated Akash address for the provider
+ * - {{.IatCurr}} - Replaced with the current timestamp
+ * - {{.Iat24h}} - Replaced with a timestamp 24 hours in the past
+ * - {{.NbfCurr}} - Replaced with the current timestamp
+ * - {{.Nbf24h}} - Replaced with a timestamp 24 hours in the past
+ * - {{.Exp48h}} - Replaced with a timestamp 48 hours in the future
+ * @param testCase - The test case containing template values
+ * @returns The test case with template values replaced
+ */
+export function replaceTemplateValues(testCase: (typeof jwtClaimsTestCases)[0]) {
+  const now = Math.floor(Date.now() / 1000);
+  const issuer = createAkashAddress();
+  const provider = createAkashAddress();
+
+  const claims = { ...testCase.claims } as any;
+  if (claims.iss === "{{.Issuer}}") claims.iss = issuer;
+  if (claims.iat === "{{.IatCurr}}") claims.iat = now;
+  if (claims.iat === "{{.Iat24h}}") claims.iat = now + 86400; // 24 hours from now
```

Comment:
suggestion(non-blocking): create a constant `ONE_DAY` or `ONE_HOUR`. Then you will be able to write

```
now + 24 * ONE_HOUR
// or
now + ONE_DAY
// or
now + 2 * ONE_DAY
```

Which is self-explanatory and doesn't require comment